rllib/tests/test_multi_agent_env.py

```diff
@@ -8,7 +8,7 @@
 
 import ray
 from ray.rllib.agents.pg import PGTrainer
-from ray.rllib.agents.pg.pg_policy import PGTFPolicy
+from ray.rllib.agents.pg.tf_pg_policy import PGTFPolicy
```

Comment:
How about we go with `<alg>_<framework>_policy` as a standard for naming, like `a3c_tf_policy`? That way the policy files always end up near each other in alphabetical sort.